logic.js

```diff
@@ -0,0 +1,64 @@
+let posts = []
```

Comment:
all those variables should be declared inside the Tweeter module - it's the one that uses them